src/test/java/com/rultor/agents/github/ReportsTest.java

```diff
@@ -47,18 +49,68 @@
  * @since 1.3
  */
 public final class ReportsTest {
+    /**
+     * Message bundle.
+     */
+    private static final ResourceBundle PHRASES =
+        ResourceBundle.getBundle("phrases");
+
+    /**
+     * Xpath to check that talk was executed correctly.
+     */
+    private static final String XPATH = "/talk[not(request)]";
 
     /**
      * Reports can report a result of a request.
-     * @throws Exception In case of error.
+     * @throws Exception In case of error
      */
     @Test
     public void reportsRequestResult() throws Exception {
         final Repo repo = new MkGithub().randomRepo();
         final Issue issue = repo.issues().create("", "");
+        final Talk talk = ReportsTest.example(repo, issue);
+        final Agent agent = new Reports(repo.github());
+        agent.execute(talk);
+        MatcherAssert.assertThat(
+            talk.read(),
+            XhtmlMatchers.hasXPath(XPATH)
+        );
+    }
+
+    /**
+     * Reports can report a result of a request, when stop command fails.
+     * @throws Exception In case of error
+     */
+    @Test
+    public void reportsRequestResultWhenStopFails() throws Exception {
+        final Repo repo = new MkGithub().randomRepo();
+        final Issue issue = repo.issues().create("Bug", "stop it please");
+        final Talk talk = ReportsTest.example(repo, issue);
         final Agent agent = new Reports(repo.github());
-        final Talk talk = new Talk.InFile();
-        talk.modify(
+        agent.execute(talk);
+        MatcherAssert.assertThat(
+            talk.read(),
+            XhtmlMatchers.hasXPath(XPATH)
+        );
+        MatcherAssert.assertThat(
+            "Comment contains warning about stop request",
+            repo.issues().get(1).comments().get(1).toString().contains(
```

Comment:
@mkordas I think current variant is better then variant with full check:

```
 MatcherAssert.assertThat(
            "Comment contains warning about stop request",
            repo.issues().get(1).comments().get(1).toString().equals(
                String.format(
                    "%s %s",
                    ReportsTest.PHRASES.getString("Reports.stop-fails"),
                    Logger.format(
                        ReportsTest.PHRASES.getString("Reports.success"),
                        "http://www.rultor.com/t/1-1",
                        1260000L
                    )
                )
            )
        );
```

Or not?